custom_components/tuya_local/devices/globe_ceilingfanrgbcwlight.yaml

```diff
@@ -0,0 +1,142 @@
+name: Ceiling Fan with RBCW Light
```

Comment:
Device names should comply with HA naming convention - only first word capitalised, with exception for abbreviations like RGBCW. So "Ceiling fan with RGBCW light"